provider-service/kfp/Makefile

```diff
@@ -0,0 +1,47 @@
+include ../../common.mk
+include ../../docker-targets.mk
+include ./get-proto.mk
+
+IMG := kfp-operator-kfp-provider-service
+
+all: build
+
+##@ Build
+
+MOCKGEN := $(PROJECT_DIR)/bin/mockgen
+mockgen: ## Download mockgen locally if necessary.
+	$(call go-install,$(PROJECT_DIR)/bin/mockgen,github.com/golang/mock/mockgen@v1.6.0)
+
+generate: protoc-gen-go mockgen ## Generate service definitions from protobuf
+	$(call get-proto,github.com/google/ml-metadata,v1.5.0)
```

Comment:
what was the reason for not using a more up-to-date version of this?

Reply:
Likely a copy from the existing provider argo workflows. For the sake of simplicity in this change I would advise not changing this version here and instead having it as a separate issue.